src/modals/AddManager.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+    Box,
+    Button,
+    CircledIcon,
+    Col,
+    Input,
+    ModalWrapper,
+    Row,
+    useModal
+} from '@impact-market/ui';
+import React from 'react';
+import RichText from '../libs/Prismic/components/RichText';
+
+
+const AddManager = () => {
+    const { handleClose } = useModal();
+    
+    const handleCancel = (event: any) => {
+        event.preventDefault();
+
+        return handleClose();
+    };
+
+
+    return (
+        <ModalWrapper maxW={30.25} padding={1.5} w="100%">
+            <CircledIcon icon="flash" large />
+            <RichText
+                content="Add Manager"
+                large
+                mt={1.25}
+                semibold
+            />
+            <RichText
+                content="The new manager will receive a notification and can immediatly start adding new beneficiaries."
+                g500
+                mt={0.5}
+                small
+            />
+            {/* <form onSubmit={handleSubmit(onSubmit)}> */}
+            <form>
+                <Box mt={1.25}>
+                    <Input
+                        name="message"
+                        placeholder="Address"
+                        rules={{ required: true }}
```

Comment:
If this field is required, use "hint" and "withError" as it is in Profile, so the field turns "red" when submitting the form with no address